tests/storage/test_monthly_active_users.py

```diff
@@ -28,7 +28,7 @@
 FORTY_DAYS = 40 * 24 * 60 * 60
 
 
-def gen_3pids(count: int):
+def gen_3pids(count: int) -> Dict[str, Any]:
```

Comment:
Oh sorry Dirk, I got the return type wrong here.

```suggestion
def gen_3pids(count: int) -> List[Dict[str, Any]]:
```